src/components/02-components/card-expreciences/card-experiences.tsx

```diff
@@ -0,0 +1,230 @@
+import { Button, Box, Paper, Typography } from '@mui/material';
```

Comment:
Please rename this component to `CardExperience` and the files to `card-experience`. The component is for a single experience, it should be singular.